unparser.gemspec

```diff
@@ -23,7 +23,7 @@ Gem::Specification.new do |gem|
 
   gem.metadata['rubygems_mfa_required'] = 'true'
 
-  gem.required_ruby_version = '>= 3.0'
+  gem.required_ruby_version = '>= 3.2'
```

Comment:
I cannot remove support for `3.0` and `3.1` right now. Not until they are EOL. I'll have to fix this via a conditional around the `Symbol#inspect` usage.